cluster-service/deploy/values.yaml

```diff
@@ -166,6 +166,10 @@ databaseAuthMethod: "az-entra"
 oidcIssuerBlobServiceUrl: ""
 # OIDC base issuer URL, e.g. https://<storage-account>.z1.web.core.windows.net/
 oidcIssuerBaseUrl: ""
+# tls certificates issuer specifies the issuer of the TLS certificate for Ingress and Kube API Server - either Self or OneCertV2-PublicCA.
+tlsCertificatesIssuer: ""
+# tls certificates enabled  indicates whether TLS certificates are enabled or disabled for Ingress and Kube API Server.
+tlsCertificatesEnabled: true
```

Comment:
If we set enabled here we should set "Self" for the issuer right? otherwise it will be invalid